corehq/apps/reports/filters/api.py

```diff
@@ -126,12 +126,12 @@ def data_sources(self):
                 (self.get_static_options_size, self.get_static_options),
                 (self.get_groups_size, self.get_groups),
                 (self.get_locations_size, self.get_locations),
-                (self.get_users_size, self.get_users),
```

Comment:
This will change the behavior for all reports that use the EMWF except for those with their own subclass.  That means it'll also affect for instance, the worker monitoring reports.  I can't remember where we landed on that, but wanted to flag that this won't be limited to the form and case reports.

Reply:
Updated so `EmwfOptionsView` returns what it used to (active users), and `EmwfOptionsViewAllUsers` returns all user data.